scripts/test/js_test.sh

```diff
@@ -7,6 +7,9 @@ then
 elif [[ ! -z "$CODECOV" ]]
 then
     export CMD="node ./node_modules/nyc/bin/nyc.js --reporter=lcovonly -R spec mocha"
+elif [[ ! -z "$WATCH" ]]
+then
+    export CMD="node ./node_modules/.bin/mocha --watch"
```

Comment:
seems odd that the base command here is different from the one below (`node ./node_modules/.bin/mocha` vs. `node ./node_modules/.bin/mocha`). any reason for that difference?